azure-pipelines-official.yml

```diff
@@ -289,8 +289,12 @@ stages:
       # https://github.com/dotnet/arcade/issues/2871 is resolved.
       enableSymbolValidation: false
       enableSourceLinkValidation: false
+      # It's important that post-build stages are depends on 'SetValidateDependency' stage instead of 'build', 
+      # since we don't want to publish validation build. 
       validateDependsOn:
         - SetValidateDependency
+      dependsOn:
```

Comment:
We set both because arcade 1.0 uses `dependsOn` and arcade 5.0 uses `validateDependsOn`, and I'm not certain why we are switching back and force.

Reply:
Also I found it's odd that AzDo doesn't complain about assigning value to an non-existing template parameter. Not sure if there's a reason behind that or just simply an oversight. @tmat Whom could I talk to about this?